app/views/search-for-your-gp.nunjucks

```diff
@@ -16,7 +24,7 @@
 </ul>
 <details class="" role="group">
   <summary data-analytics="summary" role="button" aria-controls="details-content-0" aria-expanded="false">
-    <span class="details__summary">If you haven't registered for online services</span>
+    <span class="details__summary" onmousedown="dcsMultiTrack('GPAppointment', 'Not_Registered', 'WT.dl', '121')">If you haven't registered for online services</span>
```

Comment:
I wouldn't go with Webtrends being the forefront of best practice though. This is about the separation of the html and javascript. Anyone else have an opinion on this @neilbmclaughlin @markysoft ?